plugins/swap/fu-plugin-swap.c

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (C) 2020 Richard Hughes <richard@hughsie.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#include "config.h"
+
+#include "fu-plugin-vfuncs.h"
+#include "fu-hash.h"
+
+#define FU_PLUGIN_SWAP_SOURCE_FILE	"/proc/swaps"
```

Comment:
I think it might be better to use the fwupd prefix stuff for this specifically.  It will then allow using this with the test suite and various mocked up files.

Reply:
Yup, the swap plugin is crying out for some unit tests too. Will fix.

Reply:
Unit tests added.